### DISCUSSION TITLE
Opentrack + Neuralnet freezing in DCS on higher graphics settings

### DISCUSSION THREAD

**Discussion by ethangallagher98** [opentrack/opentrack#1859] [Q&A]
If I try to run dcs on medium+ settings at 1440p in a training mission, opentrack becomes completely unusable. The software will update my head position less than once every couple seconds. If I go to low graphics settings it works perfectly fine. Is this expected behavior? My rig should be more than up to the task, I have a 3070 and a 5800X.

**Comment by DaWelter** [comment]
Given the hardware, I'd say it,s rather unexpected. Try this:
* In the NN settings, limit the number of threads. One or two should be fine.
* Select a low camera resolution. Only 640 px in width or so.
* Use the "Select Pose Net ONNX Button" and select small model
* Set "Frames Per Second" to 30

**Reply by ethangallagher98** [reply]
So I was able to find a workaround. On windows, disabling "Hardware accelerated GPU scheduling" mostly resolved the issue. The only issue I have now is that if I keep the headtracking running for 1+ hour the responsiveness starts to degrade until I restart the program.
